dynamic-simulation/dynamic-simulation-api/src/main/java/com/powsybl/dynamicsimulation/DynamicSimulationResultImpl.java

```diff
@@ -7,37 +7,47 @@
 package com.powsybl.dynamicsimulation;
 
 import java.util.Collections;
+import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 
-import com.powsybl.timeseries.StringTimeSeries;
 import com.powsybl.timeseries.TimeSeries;
 
 /**
  * @author Marcos de Miguel {@literal <demiguelm at aia.es>}
+ * @author Laurent Issertial {@literal <laurent.issertial at rte-france.com>}
  */
 public class DynamicSimulationResultImpl implements DynamicSimulationResult {
 
-    private final boolean status;
-    private final String logs;
+    private final Status status;
+    private final String statusText;
     private final Map<String, TimeSeries> curves;
```

Comment:
Just realizing by seeing the code smells that we didn't change anything for curves: shouldn't we take this opportunity to ask for `Map<String, DoubleTimeSeries>`? I don't think any other TimeSeries would make sense for a curve.